tasks/configure.yml

```diff
@@ -0,0 +1,56 @@
+---
+# ss-configure
+
+# Create api-key if not defined
+- set_fact: archivematica_src_configure_ss_api_key={{ 999999999999999999999 | random | to_uuid | hash('md5') }}
+  when: "archivematica_src_configure_ss_api_key is undefined and archivematica_src_configure_ss|bool"
+
+# Delete default SS superuser test
+- name: "Delete default SS superuser"
+  become: "yes"
+  shell: |
+    cd {{ archviematica_src_ss_app }}
+    echo "from django.contrib.auth.models import User;User.objects.get(username='test').delete()" | {{ archivematica_src_ss_virtualenv }}/bin/python manage.py shell
+  args:
+    chdir: "{{ archviematica_src_ss_app }}"
+    executable: /bin/bash
+  environment: "{{ archivematica_src_ss_environment }}"
+  when: "archivematica_src_configure_ss|bool"
+
+- name: "Create SS superuser"
+  django_manage:
+    command: > 
+      create_user 
+      --username={{ archivematica_src_configure_ss_user }}
+      --password={{ archivematica_src_configure_ss_password }} 
+      --email={{ archivematica_src_configure_ss_email }} 
+      --api-key={{ archivematica_src_configure_ss_api_key  }} 
+      --superuser
+    app_path: "{{ archviematica_src_ss_app }}"
+    virtualenv: "{{ archivematica_src_ss_virtualenv }}"
+  environment: "{{ archivematica_src_ss_environment }}"
+  when: "archivematica_src_configure_ss|bool"
+
+# dashboard-configure
+
+# Create api-key if not defined
+- set_fact: archivematica_src_configure_am_api_key={{ 999999999999999999998 | random | to_uuid | hash('md5') }}
+  when: archivematica_src_configure_am_api_key is undefined
```

Comment:
For the ss api key, the condition is ` when: "archivematica_src_configure_ss_api_key is undefined and archivematica_src_configure_ss|bool"`, but here you are not checking for `archivematica_src_configure_dashboard`

Reply:
Yes, it was done intentionally in case just SS is installed.

Reply:
I'm not getting your point, I think that the other case (deploying the dashboard, with the ss in other server), will also fail.

Thinking out loud , does it make sense to place this steps in pipeline-dbconf.yml for the dashboard, and ss-db.yml for the storage service? This way you can rely on `archivematica_src_install_am` and `archivematica_src_install_ss` to avoid this issue.